src/main/scala/com/databricks/labs/overwatch/utils/SparkSessionWrapper.scala

```diff
@@ -1,9 +1,21 @@
 package com.databricks.labs.overwatch.utils
 
+import com.databricks.labs.overwatch.utils.SparkSessionWrapper.{ parSessionsOn}
 import org.apache.log4j.{Level, Logger}
 import org.apache.spark.SparkContext
 import org.apache.spark.sql.SparkSession
 
+import java.util.concurrent.ConcurrentHashMap
+import scala.collection.JavaConverters._
+
+
+object SparkSessionWrapper {
+
+   var parSessionsOn = false
```

Comment:
please convert this var to private and create getter/setters.

Why did we wind up using this instead of the spark.conf("overwatch.parSessions")? I'm wondering if maintaining the global state of yet another var may cause other issues. Really no diff than the global sparkconf but just curious why we switched the approach.